x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_list_page/components/agent_activity_flyout.tsx

```diff
@@ -252,6 +252,8 @@ const actionNames: {
   ACTION: { inProgressText: 'Actioning', completedText: 'actioned', cancelledText: 'action' },
 };
 
+const getAction = (type?: string) => actionNames[type ?? 'ACTION'] ?? actionNames.ACTION;
```

Comment:
fix for https://github.com/elastic/kibana/issues/141599
uknown action types were not handled correctly